tests/test_dependencies.py

```diff
@@ -293,6 +293,34 @@ def test_sampling_rate(deps):
         deps.sampling_rate("non.existing")
 
 
+@pytest.mark.parametrize(
+    "files, column, dt, expected_type, expected_result",
+    [
+        ("file.wav", "format", None, str, "wav"),
+        (["file.wav"], "format", None, str, "wav"),
+        (["file.wav", "db.files.csv"], "format", None, str,  ['wav', 'csv']),
```

Comment:
This should fix the linter error:

```suggestion
        (["file.wav", "db.files.csv"], "format", None, str, ['wav', 'csv']),
```